pre-release/v1.10.0-a1 (#190)
fivetran/dbt_jira: CHANGELOG.md

# dbt_jira v1.10.0-a1

[PR #190](https://github.com/fivetran/dbt_jira/pull/190) includes the following update:

## Bug Fix

> - Fixes duplicate `issue_multiselect_history` records (same field, issue, and timestamp) being double-counted, which fanned out sprint rows in `jira__daily_sprint_issue_history`.

Reviewer: ```suggestion
- Removes duplicate `issue_multiselect_history` records to avoid fanout in `jira__daily_sprint_issue_history`.
```

Reviewer: Up to you if you wanna incorporate this suggestion -- just felt it could be a lil more concise
